src/qt/bitcoin.cpp

```diff
@@ -250,8 +252,10 @@ public Q_SLOTS:
     ClientModel *clientModel;
     BitcoinGUI *window;
     QTimer *pollShutdownTimer;
-#ifdef ENABLE_WALLET
+#if defined(ENABLE_WALLET) && defined(ENABLE_BIP70)
```

Comment:
Might be less messy to just leave the pointer here and not use it.

Reply:
I don't think I agree. Accidentally using it would not generate a compile error in that case anymore.

Reply:
Can't we just allow ENABLE_BIP70 if ENABLE_WALLET is true? In that case here `if defined(ENABLE_BIP70)` would be enough all over the place.

Reply:
That'd make sense.